packages/ihe-gateway/server/CodeTemplates/Common Library/Config/Config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This module provides a standardized way to access environment variables.
+ */
+
+function getEnvVar(name) {
+  const value = java.lang.String(Packages.java.lang.System.getenv(name));
+  if (value == null) {
+    throw new Error(`Environment variable ${name} is not defined.`);
+  }
+  return value;
+}
+
+const Config = {
+  getSamlHomeCommunityId: function () {
+    return getEnvVar("SAML_HOME_COMMUNITY_ID");
+  },
+
+  getInboundXcpdUrl: function () {
```

Comment:
Use `PD` instead of `Xcpd` since we're using `DQ` and `DR`?

Reply:
We use `XCPD` in a lot of other places as well, such as `ihe-gateway-sdk` and `external/carequality`. I think at this point we just roll with it?

Reply:
I can't speak about the other ones b/c I don't know, what I know is that we agreed to have XCA, ITI-55, XCPD, etc. only on protocol-specific code, and use the business naming PD, DQ, DR everywhere else.

If there are other places using XCPD that shouldn't, that a deprecate that needs fixing at some point.

We're building the system that we'll work with, update, maintain for years, so while its totally fine and healthy to focus on what we need for now, we need some structure, especially around names/concepts, so we can maintain a good development speed as we throw more requirements and devs into this machine.

And there's the case about consistency: let's use the same approach, in this case, since we have DQ/DR, I think it makes sense to be PD, right?